packages/checkout/components/text-input/test/validated-text-input.tsx

```diff
@@ -0,0 +1,116 @@
+/**
+ * External dependencies
+ */
+import { act, render, screen } from '@testing-library/react';
+import { VALIDATION_STORE_KEY } from '@woocommerce/block-data';
+import { dispatch, select } from '@wordpress/data';
+import userEvent from '@testing-library/user-event';
+
+/**
+ * Internal dependencies
+ */
+import { ValidatedTexInputWithoutId as ValidatedTextInput } from '../validated-text-input';
```

Comment:
Why do we need this without an instanceId? does it fail during tests?

Reply:
Yes, you will get different results depending if you run a test as part of the whole suite, or individually. Is there a way around this?